examples/Multi_Armed_Bandit/bernoulli_multi_armed_bandit.py

```diff
@@ -0,0 +1,175 @@
+# The corresponding search space is spanned by a {class}`~baybe.parameters.categorical.CategoricalParameter` describing the bandit's arms.
+
+n_arms = len(WIN_RATES)
+objective = BinaryTarget(name="clicked").to_objective()
+searchspace = CategoricalParameter(
+    name="Bandit Arm", values=[f"arm_{i}" for i in range(n_arms)]
+).to_searchspace()
+surrogate = BernoulliMultiArmedBanditSurrogate()
+
+
+# For each simulation, we report the trajectory of earned rewards and the estimated win rates of the bandit arms:
+
+
+class SimulationResult(NamedTuple):
+    """Result of a bandit campaign simulation."""
+
+    earned_rewards: np.ndarray
+    estimated_win_rates: np.ndarray
+
+
+# Putting all pieces together, we define a function to simulate our campaign under a given acquisition strategy:
+
+
+def simulate(acqf: AcquisitionFunction) -> SimulationResult:
+    """Simulate the campaign with with the given acquisition function."""
+    recommender = TwoPhaseMetaRecommender(
+        initial_recommender=RandomRecommender(
+            allow_repeated_recommendations=True,
+            allow_recommending_already_measured=True,
+        ),
+        recommender=BotorchRecommender(
+            surrogate_model=surrogate,
+            acquisition_function=acqf,
+            allow_repeated_recommendations=True,
+            allow_recommending_already_measured=True,
+        ),
+    )
+
+    # Containers for storing the rewards and win rate estimates
+    earned_rewards = np.zeros((N_MC_RUNS, N_ITERATIONS))
+    estimated_win_rates = np.zeros((N_MC_RUNS, n_arms))
+
+    for mc in range(N_MC_RUNS):
+        campaign = Campaign(searchspace, objective, recommender)
+
+        for i in range(N_ITERATIONS):
+            df = campaign.recommend(batch_size=1)
+            recommended_arm_idx = int(df.iloc[0, 0].split("_")[1])
+            reward = BANDIT.draw_arm(recommended_arm_idx)
+            earned_rewards[mc, i] = reward
+            df["clicked"] = reward
+            campaign.add_measurements(df)
+
+        estimated_win_rates[mc] = surrogate.posterior_means()
+
+    return SimulationResult(earned_rewards, estimated_win_rates)
+
+
+### Running the Simulation
+
+# With our model in place, we can now simulate the campaign for both acquisition functions.
+# The success of our two objectives is quantified using the following metrics:
+# - For online optimization, we look at the **empirically achieved win rate** and
+#   compare it to the win rate of the best arm.
+# - For active learning, we report the **mean squared error of the win rates**
+#   estimated at the end of the campaign.
+
+for acqf in ACQFS:
+    name = acqf.__class__.__name__
+    result = simulate(acqf)
+    MSE = np.linalg.norm(result.estimated_win_rates - WIN_RATES, "fro") / n_arms
+
+    print(name)
+    print("-" * len(name))
+    print("Optimal win rate:", max(WIN_RATES))
+    print("Achieved win rate:", result.earned_rewards.mean())
```

Comment:
can we also print the estimated win rates? could be interesting for the active learning case and comparison

Reply:
How? You get as many as you have MC runs. And taking the average or something is not fair/correct since that would smooth out your estimation noise.

Reply:
would the MAE per winrate be a compromise?

Reply:
I was usggesting the average error `per winrate` not the overall one

Reply:
OK, so you mean a length-5 vector? Would indeed show the effect that mis-identification rather happens for worse bandit arms. 

Suggestion: Because we use the posterior mean as estimator, I'd prefer to stick with MSE as a metric. But we could nicely split this using the bias-variance decomposition and show both. The bias part would then correspond to what you have in mind. What do you think?

Reply:
I would not add anything more to this example, in my opinion, it already has too much content (unless it's a one-liner).

Reply:
well its not really an addition, its a change to the already displayed metric, im just saying it could be more informative to show it `per arm` and not averaged over all
but this is not major do as you like

Reply:
Played around with it but also got to the impression that the output got too cluttered. But still: have change the computation such that it's done via bias-variance decomposition, so that both parts are now at least pre-computed as variables